kaldb/src/test/java/com/slack/kaldb/testlib/KaldbConfigUtil.java

```diff
@@ -74,10 +74,20 @@ public static KaldbConfigs.KaldbConfig makeKaldbConfig(
                     .build())
             .build();
 
+    KaldbConfigs.RecoveryConfig recoveryConfig =
+        KaldbConfigs.RecoveryConfig.newBuilder()
+            .setServerConfig(
+                KaldbConfigs.ServerConfig.newBuilder()
+                    .setServerPort(indexerPort + 3)
```

Comment:
This needs to not use a magic number per https://github.com/slackhq/kaldb/pull/249

Reply:
Done.